syncer/ddl.go

```diff
@@ -38,7 +38,7 @@ type parseDDLResult struct {
 
 func (s *Syncer) parseDDLSQL(sql string, p *parser.Parser, schema string) (result parseDDLResult, err error) {
 	// check skip before parse (used to skip some un-supported DDLs)
-	ignore, err := s.skipQuery(nil, nil, sql)
+	ignore, err := s.filterSQL(sql)
```

Comment:
unify `ignore`, `shouldSkip`, `filtered`

Reply:
`skip` used in so many place, so we unified use `skip`, and change function name `filter*` to `skip*`.